packages/x-components/src/__tests__/adapter.dummy.ts

```diff
@@ -1,28 +1,6 @@
-import { SearchAdapter } from '@empathyco/x-adapter';
 import { PlatformAdapter } from '@empathyco/x-adapter-platform';
 
-export const SearchAdapterDummy: SearchAdapter = {
-  // Required functions
-  getNextQueries: jest.fn(),
-  getTopRecommendations: jest.fn(),
-  getSectionRecommendations: jest.fn(),
-  getQueriesRecommendations: jest.fn(),
-  getClicksRecommendations: jest.fn(),
-  getUserRecommendations: jest.fn(),
-  getRelatedTags: jest.fn(),
-  getSuggestions: jest.fn(),
-  search: jest.fn(),
-  searchById: jest.fn(),
-  track: jest.fn(),
-
-  // Optional functions
-  invalidateCache: jest.fn(),
-  setConfig: jest.fn(),
-  addConfigChangedListener: jest.fn(),
-  removeConfigChangedListener: jest.fn()
-};
-
-export const PlatformAdapterDummy: PlatformAdapter = {
+export const SearchAdapterDummy: PlatformAdapter = {
```

Comment:
Should we rename it to `AdapterDummy` to avoid confusion with the old `SearchAdapter`?